force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +543,17 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+        // Compare the ages of the two Person objects
+        if(person1.getAge() > person2.getAge()){
+            // If person1 is older, return person1
+            return person1;
+        }else if(person1.getAge() < person2.getAge()){
+            // If person2 is older, return person2
+            return person2;
+        }else{
```

Comment:
You can simplify this conditional to just an if-else since your current if block and else block both return the same value. Just combine the conditions into the first if condition: 
`if (person1.getAge() >= person2.getAge()) {`